Mastodon/Scene/Onboarding/Welcome/View/Pages/WelcomeContentPageView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  WelcomeContentPageView.swift
+//  Mastodon
+//
+//  Created by Nathan Mattes on 26.11.22.
```

Comment:
Please use  the yyyy-mm-dd format.
https://xkcd.com/1179/

Reply:
We could also consider dropping the header in general. In its current form I don't see any value in it at all. The filename is part of, well, the filename. And the date of creation can be inferred from the Git history.

Reply:
Definitely. I'm not sure what the status of copyright assignment is in this project, but *if* you want to have file-level headers this is the recommended way: https://reuse.software/faq/#step-2